.github/workflows/windows-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+name: e2e tests on Windows
+
+on:
+  schedule:
+    - cron: '13 5 * * 1-5'
+  workflow_dispatch: {}
+defaults:
+  run:
+    shell: powershell
+jobs:
+
+  e2e-tests:
+    timeout-minutes: 90
+    runs-on: [self-hosted, Windows, X64]
+    steps:
+      - run: wsl --unregister rancher-desktop
+        continue-on-error: true
+      - run: wsl --unregister rancher-desktop-data
+        continue-on-error: true
+      - run: >
+          Remove-Item -Path "~/AppData/Local/rancher-desktop" -Recurse -Force -ErrorAction Ignore
+      - run: >
+          Remove-Item -Path "~/AppData/Roaming/rancher-desktop" -Recurse -Force -ErrorAction Ignore
+      - name: Remove golang module cache
+        run: Remove-Item -Path ~/go/pkg/mod -Recurse -Force -ErrorAction Ignore
```

Comment:
please, add quotes to the path to match other Remove-Item commands

Reply:
Will do. (There are no spaces here, so it's not necessary, but it's a good habit)